05-single-hop-route/task02.py

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env python3
+import sys
+import os
+import argparse
+sys.path.append("../testutils")
+
+from testutils import Board
+from iotlab import create_experiment, stop_experiment, get_nodes_addresses
+from common import SingleHopNode, single_hop_run, print_results
+
+p = argparse.ArgumentParser()
+p.add_argument('riotbase', nargs='?',
+               help='Location of RIOT folder')
+args = p.parse_args()
+riotbase = args.riotbase
+
+if not riotbase:
+    p.print_help()
+    sys.exit(1)
+
+os.chdir(os.path.join(riotbase, "tests/gnrc_udp"))
+
+#Create IoTLAB experiment (TODO: Return addresses)
```

Comment:
I think the TODO would make the function more complicated than it needs to be. Keeping it seperated within `get_nodes_addresses()` makes the API simpler.